src/EditorFeatures/CSharpTest/UseInferredMemberName/UseInferredMemberNameTests.cs

```diff
@@ -110,7 +110,7 @@ class C
     void M()
     {
         int a = 1;
-        var t = new { [||]a=a, 2 };
+        var t = new { [||]a= a, 2 };
```

Comment:
Why the change here?

Reply:
Extra trivia in the input should be considered elastic, so the output should not be affected in this case.